src/vm/amd64/calldescrworkeramd64.S

```diff
@@ -42,8 +42,12 @@ NESTED_END FastCallFinalizeWorker, _TEXT
 
 NESTED_ENTRY CallDescrWorkerInternal, _TEXT, CallDescrWorkerUnwindFrameChainHandler
         push_nonvol_reg rbx             // save nonvolatile registers
+        .cfi_def_cfa_offset 16          //
```

Comment:
The .cfi annotations should be in the prolog setup/teardown macros. Something like:

.macro push_nonvol_reg Register
        push \Register
        .cfi_adjust_cfa_offset 8
        .cfi_offset \Register, 0
.endm

Similar for others. Ideally, we should not have any direct use of .cfi annotations - we may need to introduce a few macros to make it work (e.g. pop_nonvol_reg).